R/build-home-index.R

```diff
@@ -37,19 +43,114 @@ data_home <- function(pkg = ".") {
   ))
 }
 
+
 data_home_sidebar <- function(pkg = ".") {
+
   pkg <- as_pkgdown(pkg)
-  if (!is.null(pkg$meta$home$sidebar))
+  if (isFALSE(pkg$meta$home$sidebar))
     return(pkg$meta$home$sidebar)
 
-  paste0(
-    data_home_sidebar_links(pkg),
-    data_home_sidebar_license(pkg),
-    data_home_sidebar_community(pkg),
-    data_home_sidebar_citation(pkg),
-    data_home_sidebar_authors(pkg),
-    collapse = "\n"
+  html_path <- file.path(pkg$src_path, pkg$meta$home$sidebar$html)
+
+  if (length(html_path)) {
+
+    if (!file.exists(html_path)) {
+      abort(
+        sprintf(
+          "Can't find file '%s' specified by %s.",
+          pkg$meta$home$sidebar$html,
+          pkgdown_field(pkg = pkg, "home", "sidebar", "html")
+          )
```

Comment:
Can you please fix these trailing `)`? The current indenting really confuses my eye